### DISCUSSION TITLE
how to use detect.py without notebook.

### DISCUSSION THREAD

**Discussion by helloankur** [ultralytics/yolov5#4584] [Q&A]
How to use the yolov5 train model from IDE without using a notebook like a google collab, Kaggle notebook, etc ?
is there any way to use pycharm or VStudio etc?

**Comment by glenn-jocher** [comment]
@helloankur I don't understand your question. PyCharm and VStudio are the default use cases. Nothing special is required to use them.

<img width="1347" alt="Screenshot 2021-08-29 at 13 52 39" src="https://user-images.githubusercontent.com/26833433/131249430-ab5748bb-22a6-4c06-ab26-2c7f3064d6a2.png">

**Reply by orphan2020** [reply]
Please @glenn-jocher show some mercy with us that aren't pros ☺️🙏

**Reply by glenn-jocher** [reply]
@orphan2020 don't worry, it's really easy. You just install PyCharm, and then pick a GitHub repo to clone and start from. It does everything for you.
https://www.jetbrains.com/help/pycharm/installation-guide.html

<img width="1117" alt="Screenshot 2021-08-29 at 14 25 48" src="https://user-images.githubusercontent.com/26833433/131250277-1cfd51ad-789b-42ac-a794-f26ded167b46.png">
